code/game/objects/structures/kitchen_spike.dm

```diff
@@ -25,55 +48,58 @@
 /obj/structure/kitchenspike/attack_paw(mob/user)
 	return src.attack_hand(usr)
 
-/obj/structure/kitchenspike/attackby(obj/item/weapon/grab/G, mob/user, params)
-	if(!istype(G, /obj/item/weapon/grab))
-		return
-	if(istype(G.affecting, /mob/living/carbon/monkey))
-		if(src.occupied == 0)
-			src.icon_state = "spikebloody"
-			src.occupied = 1
-			src.meat = 5
-			src.meattype = MEATTYPE_MONKEY
-			src.skin = 1
-			src.skintype = SKINTYPE_MONKEY
-			for(var/mob/O in viewers(src, null))
-				O.show_message(text("<span class='danger'>[user] has forced [G.affecting] onto the spike, killing them instantly!</span>"))
-			qdel(G.affecting)
-			qdel(G)
-
+/obj/structure/kitchenspike/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/crowbar))
+		if(!src.occupied)
+			playsound(loc, 'sound/items/Crowbar.ogg', 100, 1)
+			if(do_after(user, 20, target = src))
+				user << "<span class='notice'>You pry the spikes out of the frame.</span>"
+				new /obj/item/stack/rods(loc, 4)
+				var/obj/F = new /obj/structure/kitchenspike_frame(src.loc,)
+				transfer_fingerprints_to(F)
+				qdel(src)
 		else
-			user << "<span class='danger'>The spike already has something on it, finish collecting its meat first!</span>"
-	else if(istype(G.affecting, /mob/living/carbon/alien))
-		if(src.occupied == 0)
-			src.icon_state = "spikebloodygreen"
-			src.occupied = 1
-			src.meat = 5
-			src.meattype = MEATTYPE_ALIEN
-			src.skin = 1
-			src.skintype = SKINTYPE_ALIEN
-			for(var/mob/O in viewers(src, null))
-				O.show_message(text("<span class='danger'>[user] has forced [G.affecting] onto the spike, killing them instantly!</span>"))
-			qdel(G.affecting)
-			qdel(G)
+			user << "<span class='notice'>You can't do that while something's on the spike!</span>"
+	else if(istype(I, /obj/item/weapon/grab))
+		var/obj/item/weapon/grab/G = I
+		if(istype(G.affecting, /mob/living/carbon/monkey) || istype (G.affecting, /mob/living/carbon/alien) || istype (G.affecting, /mob/living/simple_animal/hostile/bear) || istype (G.affecting, /mob/living/simple_animal/pet/dog/corgi))
+			if(!src.occupied)
+				src.occupied = 1
+				src.meat = 5
+				src.skin = 1
+				if(istype(G.affecting, /mob/living/carbon/monkey))
+					src.meattype = MEATTYPE_MONKEY
+					src.skintype = SKINTYPE_MONKEY
+					src.icon_state = "spikebloody"
+				if(istype(G.affecting, /mob/living/carbon/alien))
+					src.icon_state = "spikebloodygreen"
+					src.meattype = MEATTYPE_ALIEN
+					src.skintype = SKINTYPE_ALIEN
+				if(istype(G.affecting, /mob/living/simple_animal/hostile/bear))
+					src.icon_state = "spikebloodybearz"
+					src.meattype = MEATTYPE_BEAR
+					src.skintype = SKINTYPE_BEAR
+				if(istype(G.affecting, /mob/living/simple_animal/pet/dog/corgi))

```

Comment:
oh my god four if(istype())s in a row I am having horrifying flashbacks to pipe construction code